charts/scalardl/values.yaml

```diff
@@ -19,12 +19,14 @@ envoy:
   envoyConfiguration:
     # -- admin log path
     adminAccessLogPath: /dev/stdout
+    # -- list of service name and port
+    serviceListeners: scalardl-service:50051,scalardl-privileged:50052
 
-  image:
+    image:
```

Comment:
It looks like an unnecessary indentation.

Reply:
@superbrothers
 
I apologize for the inconvenience. Thank you very much. :bow:
I've corrected it.

https://github.com/scalar-labs/helm-charts/pull/36/commits/90a7172e57bd9216403996feb5c16f3b47c549e5